web/src/engine/websites/GalaxyManga.ts

```diff
@@ -0,0 +1,90 @@
+import { Tags } from '../Tags';
+import icon from './GalaxyManga.webp';
+import { Chapter, DecoratableMangaScraper, Manga, Page, type MangaPlugin } from '../providers/MangaPlugin';
+import * as Common from './decorators/Common';
+import { FetchJSON, FetchRequest } from '../FetchProvider';
+
+type APIManga = {
+    status: boolean,
+    message: string,
+    serie: {
+        id: number,
+        title:string
+    }
+}
+
+type APIMangas = {
+    data: {
+        id: number,
+        title: string
+    }[]
+}
+
+type APIChapter = {
+    id: number,
+    title: string,
+    name: string
+}
+
+type APIPages = {
+    chapter: {
+        chapterData: {
+            webtoon: string[]
+        }
+    }
+}
+
+@Common.ImageAjax()
+export default class extends DecoratableMangaScraper {
+
+    private readonly apiurl = 'https://api.flixscans.com';
```

Comment:
May append `/api/v1/webtoon`

Reply:
Not a good idea

`return data.chapter.chapterData.webtoon.map(image => new Page(this, chapter, new URL(`/storage/${image}`, this.apiurl)));`